convertor/csv2format.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import datetime
+import sys
+
+csvs = sys.argv[1].split(',')
+format = sys.argv[2]
+out = sys.argv[3]
+
+def writePYSEC(mas, startIndex):
+    file_name = 'PYSEC-2022-'+ startIndex
+    f = open(out+file_name, 'a+')
```

Comment:
- use wit block:

```
with open(out+file_name, 'w') as f:
     f.write...
```

- let a filename will be just `title.pysec`
- use string interpolation f```{title}...

Reply:
fixed